### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

File: modals/login.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    border: none !important;
+    cursor: pointer !important;
+    
+    width: auto !important;
+    padding: 0.5rem 1rem!important;
+    background-color: rgba(255, 0, 0, 0.589) !important;
+    position: relative;
+    top:0rem;
+    left:0px;
+    /* position: absolute; */
```
Comment (sri-rishi) [inline modals/login.css]: remove these comments

File: modals/login.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  .login-btns:hover{
+      background-color: var(--primary-color);
+    
+
+  }
+  .cancel-btn{
+    
+    
+    padding: 14px 20px !important;
+    margin: 8px 0 !important;
```
Comment (sri-rishi) [inline modals/login.css]: Don't use !important it's bad practice

File: list/list.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    flex-direction: row;
+    justify-content: space-between;
+    padding:5px;
+    border-radius:5px;
+    
+
+}
+.list-image img{
+    width:100px;
+    height: 100px;
```
Comment (sri-rishi) [inline list/list.css]: use rems

File: list/listmain.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+*{
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
+.list-container-main{
+    
+
```
Comment (sri-rishi) [inline list/listmain.css]: remove this empty class

File: list/listmain.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    gap:1rem;
+    border-radius: 10px;
+    
+}
+.list-item-simple{
+    border:1px solid black;
+    padding:5px;
+    width:60%;
+
+
```
Comment (sri-rishi) [inline list/listmain.css]: remove extra spaces

File: modals/login.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    left:0px;
+    width: 100% !important;
+  }
+  .submit-btn:hover {
+    opacity: 0.8;
+    background-color:var(--primary-color) ;
+  }
+  .imgcontainer {
+    text-align: center;
+    margin: 24px 0 12px 0;
```
Comment (sri-rishi) [inline modals/login.css]: use shorthand property like margin: 24px 0 12px

File: modals/login.css [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+  }
+  
+  img.avatar {
+    width: 40%;
+    border-radius: 50%;
+  }
+  
+  .container-form {
+    padding: 16px;
+    background-color: aliceblue;
```
Comment (sri-rishi) [inline modals/login.css]: use variable color naming

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by sri-rishi — APPROVED** [review]

**Review by Rishabh-078 — COMMENTED** [review]
The code looks fine. You can merge it

**Review by Rishabh-078 — APPROVED** [review]
The code looks good. You can merge

**Review by swstica — COMMENTED** [review]
Modal is missing in sidebar.  Rest all looks good!

**Review by swstica — APPROVED** [review]

**Pull request merged**